pom.xml

```diff
@@ -85,70 +85,10 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
-    <dependency>
-      <groupId>com.google.http-client</groupId>
-      <artifactId>google-http-client</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-api</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-core-grpc</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.api.grpc</groupId>
-      <artifactId>proto-google-common-protos</artifactId>
-    </dependency>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-spanner</artifactId>
```

Comment:
The version that is used here will be determined by the version defined at line 72 for the import of `google-cloud-spanner-bom` (currently 2.0.2). That is the version of `google-cloud-spanner` that anyone using `google-cloud-spanner-jdbc` should be using. But:
* Any application that adds only `google-cloud-spanner-jdbc` (version 1.17.3) as a dependency without importing the `libraries-bom` will get `google-cloud-spanner` version 2.0.2.
* Any application that adds `google-cloud-spanner-jdbc` as a dependency and imports the `libraries-bom`, will get the versions of `google-cloud-spanner-jdbc` and `google-cloud-spanner` that are defined in the `libraries-bom`. The version of `google-cloud-spanner` in the BOM can be different from the one defined in `google-cloud-spanner-jdbc`.